### DISCUSSION TITLE
[IPv6] IPv6 路由优先支持

### DISCUSSION THREAD

**Discussion by cxw620** [EasyTier/EasyTier#493] [Q&A]
如题, 部分情况下, v6 路由比 v4 路由往往更佳, 但是观察到目前版本底层通过 v4 连接到其他机器后就不再尝试通过 v6 建立连接了, 不清楚是我配置问题还是不支持.

**Comment by lonnietc** [comment]
I too am greatly interested in IPv6 support and was not aware that EasyTier does not currently support it which could be a problems for my particular use case. Thanks for bringing this up for discussion, my friend.
